src/index.js

```diff
@@ -0,0 +1,38 @@
+// import _ from 'lodash';
+import './style.css';
+
+// grab items
+const todolist = document.querySelector('.to-do-list');
+
+const toDoItems = [
+  {
+    index: 1,
+    description: 'Read the novel',
+    completed: false,
+  },
+  {
+    index: 2,
+    description: 'Play video game with friends',
+    completed: false,
+  },
+  {
+    index: 3,
+    description: 'Go for a movie',
+    completed: false,
+  },
+];
+
+const dynamic = () => {
+  toDoItems.forEach((list) => {
+    todolist.innerHTML += `
+        <li class="the-item" id="${list.index}">
+            <div class="item-list">
+                <input type="checkbox">
+                <p class="item">${list.description}</p>
+            </div>
+        </li>
+        `;
+  });
+};
```

Comment:
- [ ] According to the project requirement  On page load render the dynamically created list of tasks in the dedicated placeholder, that is well implemented :clap: And according to the same project requirement, the list should appear in order of the index values for each task. But when I tried to change the index value the position of every task it still the same. Please make sure to fix it in order to make your project works as expected :+1: